os/drivers/serial/Kconfig

```diff
@@ -531,6 +537,10 @@ config SERIAL_OFLOWCONTROL
 	bool
 	default n
 
+config SERIAL_DMA
+	bool
```

Comment:
Currently, there is no DMA support in TizenRT. Do you intend to add the support.